grammars/gfm.cson

```diff
@@ -242,7 +242,8 @@
       '0':
         'name': 'support.gfm'
     'name': 'markup.code.gfm'
-    'patterns': [
+    'contentName': 'embedded.text.md'
+   'patterns': [
```

Comment:
You outdented this line by one space which will cause issues since CSON is indentation-based.